requirements.txt

```diff
@@ -3,7 +3,7 @@ pandas>=0.24.2
 numpy>=1.18.4
 dash>=1.16.2
 dash-bootstrap-components>=0.10.6
-dash-leaflet>=0.1.4
```

Comment:
Was it the reason why risk map was 'disabled' ?

Reply:
It's difficult to say certainly because the callbacks do not output error messages outside of the debugging/development mode of Dash. But I could recreate the issue locally by uninstalling `dash-leaflet` and running `pip install -r requirements.txt`. When coming back to the 0.1.4 version, the problem was solved: so, my guess is that this should work better with a `==`!